### PR TITLE
Created First two sections

### PR DESCRIPTION
**Pull request by mrprotocoll**
Add Linters
Add Index file
Add Assets (Images and CSS)
Add Styling
Add gitignore
Update README

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <a href="#" class="cta"><h3>LET’S CONNECT</h3></a>
+            <ul class="social">
+                <li class="icon"><a href="#"><img src="assets/images/twitter.svg" alt=""></a></li>
+                <li class="icon"><a href="#"><img src="assets/images/linkedin.svg" alt=""></a></li>
+                <li class="icon"><a href="#"><img src="assets/images/microverse.svg" alt=""></a></li>
+                <li class="icon"><a href="#"><img src="assets/images/github.svg" alt=""></a></li>
+                <li class="icon"><a href="#"><img src="assets/images/vector.svg" alt=""></a></li>
+            </ul>
+        </div>
```
Comment (OyePriscilla) [inline index.html]: - [x] Kindly use` section tag `to wrap all these elements, a `section` is a semantic element for creating standalone sections in a web page. It's more preferable that `div tag` to group similar elements. ℹ️

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @mrprotocoll  😃, 

Good job so far!
There is just one issue that you still need to work on to prepare your project for the final evaluation but you are almost there!

so-close-so-close-seth-meyers

## Highlights
Github flow observed✔️
Linters checks passed ✔️ 
App works correctly ✔️ 

## Required Changes ♻️

- [ ] Kindly check the inline comments below.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (mrprotocoll) [reply index.html]: Thank you @OyePriscilla. I actually used the `nav` and `header` semantics to group each navbar and hero section. I didn't think it is necessary to use a section inside those semantics. I will make the changes and create another pull request.

Comment (mrprotocoll) [reply index.html]: Done! Kindly check if it is ok with you now @OyePriscilla. so I can proceed to submit a new review.

**Review by amejid — APPROVED** [review]
Hi @mrprotocoll, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Professional README :heavy_check_mark: 
- Good commit messages :fire: 
- Descriptive PR :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (OyePriscilla) [reply index.html]: > Thank you @OyePriscilla. I actually used the `nav` and `header` semantics to group each navbar and hero section. I didn't think it is necessary to use a section inside those semantics. I will make the changes and create another pull request.

@mrprotocoll It is awesome! Actually `section` is semantic but `div `has no meaning.

**Pull request merged**